source/settings/plugin/fts-plugin.rst

```diff
@@ -344,34 +336,50 @@ Settings
    - The asterisk can only be used at the end of the header name.
      Prefix and infix usage of asterisk are not supported.
 
-   Example:
+   Example::
 
-   .. code-block:: none
+     fts_header_excludes {
+       Received = yes
+       DKIM-* = yes
+       X-* = yes
+       Comments = yes
+     }
 
-     plugin {
-       fts_header_excludes = Received DKIM-* X-* Comments
-       fts_header_includes = X-Spam-Status Comments
+     fts_header_includes {
+       X-Spam-Status = yes
+       Comments = yes
      }
 
    - ``Received`` headers, all ``DKIM-`` headers and all ``X-`` experimental
      headers are excluded, with the following exceptions:
-
-     - ``Comments`` and ``X-Spam-Status`` are indexed anyway, as they match
-       **both** ``excludes`` and ``includes`` lists.
-     - All other headers are indexed.
+   - ``Comments`` and ``X-Spam-Status`` are indexed anyway, as they match
+     **both** ``excludes`` and ``includes`` lists and ``fts_header_includes``
+     takes precedence.
+   - All other headers are indexed.
 
    Example::
 
-     plugin {
-       fts_header_excludes = *
-       fts_header_includes = From To Cc Bcc Subject Message-ID In-* X-CustomApp-*
+     fts_header_excludes {
+       * = yes
+     }
+
+     fts_header_includes {
+       From = yes
+       To = yes
```

Comment:
It's not really necessary to change these two lines to the explicit =yes format. The single line string works as well, and in this case I think it would be simpler also.

Reply:
Actually, with the current codebase from `feature/config-rewrite`, it seems it is
if I try to remove the `= yes` or just only the `yes`, I get:
```
Fatal: Error in configuration file .../dovecot.conf line xx: Expecting '{')
```
I remember we had a discussion about having yes as the default in boolean lists, but the decision seemed to be to leave it as it is (with the mandatory = yes/no part)

Reply:
Right, but I meant you could have kept it as the original syntax: `fts_header_includes = from to ...` - this works as well, and seems a bit easier to also read/write when it's trivial enough.

Reply:
The change too boolean lists was requested in the original text of DOV-6304.
As header names can also be long I would lean to this multi-line syntax
Besides, it also makes code a bit cleaner.
Marking this as resolved for now unless I have missed some points.